
From: Adrian Bunk <bunk@fs.tum.de>

  CC      drivers/net/hamachi.o
drivers/net/hamachi.c: In function `hamachi_interrupt':
drivers/net/hamachi.c:562: sorry, unimplemented: inlining failed in call 
to 'hamachi_rx': function body not available
drivers/net/hamachi.c:1402: sorry, unimplemented: called from here
make[2]: *** [drivers/net/hamachi.o] Error 1

Signed-off-by: Adrian Bunk <bunk@fs.tum.de>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/net/hamachi.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN drivers/net/hamachi.c~net-hamachic-remove-bogus-inline-at-function-prototype drivers/net/hamachi.c
--- 25/drivers/net/hamachi.c~net-hamachic-remove-bogus-inline-at-function-prototype	Thu Jul 29 14:58:13 2004
+++ 25-akpm/drivers/net/hamachi.c	Thu Jul 29 14:58:13 2004
@@ -559,7 +559,7 @@ static void hamachi_tx_timeout(struct ne
 static void hamachi_init_ring(struct net_device *dev);
 static int hamachi_start_xmit(struct sk_buff *skb, struct net_device *dev);
 static irqreturn_t hamachi_interrupt(int irq, void *dev_instance, struct pt_regs *regs);
-static inline int hamachi_rx(struct net_device *dev);
+static int hamachi_rx(struct net_device *dev);
 static inline int hamachi_tx(struct net_device *dev);
 static void hamachi_error(struct net_device *dev, int intr_status);
 static int hamachi_close(struct net_device *dev);
_
